client/front.js

```diff
@@ -0,0 +1,52 @@
+const postController = require("../controllers/post");
+
+document.getElementById('publish').addEventListener('submit', createPost(e));
```

Comment:
I assume this is the start of the process of pushing your sql data to the front end? does it work?